connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -702,10 +720,7 @@ ble_error_t Gap::acceptConnectionParametersUpdate(
         maxConnectionEventLength.value()
     );
 }
-#endif
-
 
-#if BLE_FEATURE_CONNECTABLE
```

Comment:
Same as above.

Reply:
this is already in a the connectable ifdef, look below